src/v/security/scram_credential.h

```diff
@@ -11,12 +11,14 @@
 #pragma once
 #include "bytes/bytes.h"
 #include "reflection/adl.h"
+#include "serde/envelope.h"
 
 #include <iosfwd>
 
 namespace security {
 
-class scram_credential {
+class scram_credential
+  : public serde::envelope<scram_credential, serde::version<0>> {
```

Comment:
was this commit intended to be dropped?

Reply:
it is needed, should i make it a separate PR ?